source/install/verify-signatures/disk-images.txt

```diff
@@ -0,0 +1,43 @@
+.. _compass-verify-signatures-disk-image:
+
+============================================
+Verify Packages with Disk Image Verification
+============================================
+
```

Comment:
(spacing)
There's an extra line between the title and the first para, in the [HTML](https://preview-mongodbjeffallenmongo.gatsbyjs.io/compass/DOCSP-38230/install/verify-signatures/disk-images/).

Reply:
Hmm I don't think I'm seeing this in my staged view? Could you maybe show a screenshot?

Reply:
@sarah-olson-mongodb I can't see anything off with the rst here, so I'm going to chalk this up to an odd staging behavior.